fs/romfs/Kconfig

```diff
@@ -20,4 +20,10 @@ config FS_ROMFS_CACHE_NODE
 		is mounted so that we can quick access entry of ROMFS
 		filesystem on emmc/sdcard.
 
+config FS_ROMFS_CACHE_FILE_NSECTORS
+	int "The number of file cache sector"
+	default 1
```

Comment:
do we need to add range check here so 0 can't be set?

Reply:
Done. set range to 1-256